### PR TITLE
[bg] Add translations, fix errors and typos

### PR DESCRIPTION
**Pull request by kristiyanivanovx**

### CODE REVIEW

File: locales/bg/bg.json [hunk 1]
```diff
@@ -475,26 +475,26 @@
     "Run Action": "Изпълни действие",
     "Russian Federation": "Русия",
     "Rwanda": "Руанда",
-    "Réunion": "Réunion",
+    "Réunion": "Реюнион",
     "Saint Barthelemy": "Свети Вартоломей",
-    "Saint Barthélemy": "Saint Barthélemy",
+    "Saint Barthélemy": "Сен Бартелми",
```
Comment (caouecs) [inline locales/bg/bg.json:480]: Why is it different with previous tag ?

File: locales/bg/bg.json [hunk 2]
```diff
@@ -475,26 +475,26 @@
...
     "Saint Helena": "Остров Света Елена",
     "Saint Kitts And Nevis": "Сейнт Китс и Невис",
-    "Saint Kitts and Nevis": "Saint Kitts and Nevis",
+    "Saint Kitts and Nevis": "Федерация Сейнт Китс и Невис",
```
Comment (caouecs) [inline locales/bg/bg.json:483]: Why is it different with previous tag ?

File: locales/bg/bg.json [hunk 3]
```diff
@@ -475,26 +475,26 @@
...
     "Saint Lucia": "Сейнт Лусия",
     "Saint Martin": "Свети Мартин",
-    "Saint Martin (French part)": "Saint Martin (French part)",
+    "Saint Martin (French part)": "Свети Мартин (френска част)",
     "Saint Pierre And Miquelon": "Сен Пиер и Микелон",
-    "Saint Pierre and Miquelon": "Saint Pierre and Miquelon",
+    "Saint Pierre and Miquelon": "Сен Пиер и Микелон",
     "Saint Vincent And Grenadines": "Сейнт Винсент и Гренадини",
-    "Saint Vincent and the Grenadines": "Saint Vincent and the Grenadines",
+    "Saint Vincent and the Grenadines": "Сейнт Винсент",
```
Comment (caouecs) [inline locales/bg/bg.json:490]: Why is it different with previous tag ?

File: locales/bg/bg.json [hunk 4]
```diff
@@ -475,26 +475,26 @@
...
     "Samoa": "Самоа",
     "San Marino": "Сан Марино",
     "Sao Tome And Principe": "Сао Томе и Принсипи",
-    "Sao Tome and Principe": "Sao Tome and Principe",
+    "Sao Tome and Principe": "Демократична република Сао Томе и Принсипи",
```
Comment (caouecs) [inline locales/bg/bg.json:494]: Why is it different with previous tag ?

File: locales/bg/bg.json [hunk 5]
```diff
@@ -541,7 +541,7 @@
     "Sudan": "Судан",
     "Suriname": "Суринам",
     "Svalbard And Jan Mayen": "Свалбард и Ян Майен",
-    "Svalbard and Jan Mayen": "Svalbard and Jan Mayen",
+    "Svalbard and Jan Mayen": "Ян Майен",
```
Comment (caouecs) [inline locales/bg/bg.json:544]: Why is it different with previous tag ?

File: locales/bg/bg.json [hunk 6]
```diff
@@ -626,16 +626,16 @@
...
     "Tonga": "Тонга",
     "Too Many Requests": "Твърде много заявки",
-    "total": "ваш",
-    "Total:": "Total:",
+    "total": "обща сума",
+    "Total:": "Обща сума:",
     "Trashed": "Победен",
     "Trinidad And Tobago": "Тринидад и Тобаго",
-    "Trinidad and Tobago": "Trinidad and Tobago",
+    "Trinidad and Tobago": "Република Тринидад и Тобаго",
```
Comment (caouecs) [inline locales/bg/bg.json:633]: Why is it different with previous tag ?

File: locales/bg/bg.json [hunk 7]
```diff
@@ -626,16 +626,16 @@
...
     "Tunisia": "Тунис",
     "Turkey": "Пуйка",
     "Turkmenistan": "Туркменистан",
     "Turks And Caicos Islands": "Островите Търкс и Кайкос",
-    "Turks and Caicos Islands": "Turks and Caicos Islands",
+    "Turks and Caicos Islands": "Търкс и Кейкос",
```
Comment (caouecs) [inline locales/bg/bg.json:638]: Why is it different with previous tag ?

**Review by caouecs — CHANGES_REQUESTED** [review]

File: locales/bg/bg.json (hunk 1 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:480]: Both translations are valid, it is both "Свети Вартоломей" and "Сен Бартелми", though one could argue that "Свети Вартоломей" is more localized and "Сен Бартелми" is close to the original name.

File: locales/bg/bg.json (hunk 2 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:483]: Both translations are valid. For example, if a country is a federation, as Saint Kitts and Nevis is, in Bulgarian it is both valid to say "Сейнт Китс и Невис" , meaning "Saint Kitts and Nevis" and "Федерация Сейнт Китс и Невис", meaning "Federation of Saint Christopher and Nevis".

File: locales/bg/bg.json (hunk 3 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:490]: You could say "Сейнт Винсент и Гренадини" or just "Сейнт Винсент". Source: https://bg.wikipedia.org/wiki/%D0%A1%D0%B5%D0%B9%D0%BD%D1%82_%D0%92%D0%B8%D0%BD%D1%81%D0%B5%D0%BD%D1%82_%D0%B8_%D0%93%D1%80%D0%B5%D0%BD%D0%B0%D0%B4%D0%B8%D0%BD%D0%B8

File: locales/bg/bg.json (hunk 4 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:494]: "Демократична република Сао Томе и Принсипи" means "Democratic Republic of São Tomé and Príncipe".

File: locales/bg/bg.json (hunk 6 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:633]: The second means "Republic of Trinidad and Tobago", the official naming.

File: locales/bg/bg.json (hunk 7 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:638]: The second ones doesn't have the word "Островите" which means "the islands", "Търкс и Кейкос" means "Turks and Caicos" only.

File: locales/bg/bg.json (hunk 5 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:544]: "Ян Майен" means just "Jan Mayen", the name of the island.

File: locales/bg/bg.json (hunk 1 above)
Comment (andrey-helldar) [reply locales/bg/bg.json:480]: This is a feature of Laravel projects - they don't always use the same spelling. Therefore, we indicate all the keys for transfers.
I recommend using the same translation for both keys.

File: locales/bg/bg.json (hunk 2 above)
Comment (andrey-helldar) [reply locales/bg/bg.json:483]: This is a feature of Laravel projects - they don't always use the same spelling. Therefore, we indicate all the keys for transfers.
I recommend using the same translation for both keys.

File: locales/bg/bg.json (hunk 3 above)
Comment (andrey-helldar) [reply locales/bg/bg.json:490]: This is a feature of Laravel projects - they don't always use the same spelling. Therefore, we indicate all the keys for transfers.
I recommend using the same translation for both keys.

File: locales/bg/bg.json (hunk 4 above)
Comment (andrey-helldar) [reply locales/bg/bg.json:494]: This is a feature of Laravel projects - they don't always use the same spelling. Therefore, we indicate all the keys for transfers.
I recommend using the same translation for both keys.

File: locales/bg/bg.json (hunk 5 above)
Comment (andrey-helldar) [reply locales/bg/bg.json:544]: This is a feature of Laravel projects - they don't always use the same spelling. Therefore, we indicate all the keys for transfers.
I recommend using the same translation for both keys.

File: locales/bg/bg.json (hunk 6 above)
Comment (andrey-helldar) [reply locales/bg/bg.json:633]: This is a feature of Laravel projects - they don't always use the same spelling. Therefore, we indicate all the keys for transfers.
I recommend using the same translation for both keys.

File: locales/bg/bg.json (hunk 7 above)
Comment (andrey-helldar) [reply locales/bg/bg.json:638]: This is a feature of Laravel projects - they don't always use the same spelling. Therefore, we indicate all the keys for transfers.
I recommend using the same translation for both keys.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]
@kristiyanivanovx, please choose the most correct translation for both options.

Key duplication is a feature of the Laravel projects we are trying to flatten out.

File: locales/bg/bg.json (hunk 7 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:638]: @andrey-helldar  I'm not sure I understand what you meant by "Therefore, we indicate all the keys for transfers.". Did you mean that a custom translation should be made per project, and the key duplication is an option for that?

Comment (andrey-helldar) [reply locales/bg/bg.json:638]: In different Laravel projects you can find the following usage of names

- Turks And Caicos Islands
- Turks and Caicos Islands

From a programming point of view, these are different names. But we translate them the same way. In this case, you suggest using this option:

- Turks And Caicos Islands: Островите Търкс и Кайкос
- Turks and Caicos Islands: Търкс и Кейкос

I suggest using one translation option for both keys:

- Turks And Caicos Islands: Островите Търкс и Кайкос
- Turks and Caicos Islands: Островите Търкс и Кайкос

or

- Turks And Caicos Islands: Търкс и Кейкос
- Turks and Caicos Islands: Търкс и Кейкос

Comment (andrey-helldar) [reply locales/bg/bg.json:638]: I do not know which name would be more correct in Bulgarian, therefore I suggest using a more correct version for both translation keys.

File: locales/bg/bg.json (hunk 1 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:480]: Updated accordingly.

File: locales/bg/bg.json (hunk 2 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:483]: Updated accordingly.

File: locales/bg/bg.json (hunk 3 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:490]: Updated accordingly.

File: locales/bg/bg.json (hunk 4 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:494]: Updated accordingly.

File: locales/bg/bg.json (hunk 5 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:544]: Updated accordingly.

File: locales/bg/bg.json (hunk 6 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json:633]: Updated accordingly.

**Comment by kristiyanivanovx** [comment]
I have updated translations based on your feedback.

File: locales/bg/bg.json (hunk 7 above)
Comment (kristiyanivanovx) [reply locales/bg/bg.json]: Got it. Now that I think about it, your suggestion is more suitable. Skipping or adding words changes the meaning of the phrases.

**Review by andrey-helldar — APPROVED** [review]

**Comment by caouecs** [comment]
Thank you

**Comment by kristiyanivanovx** [comment]
@reppair it do be like that, lol

**Comment by andrey-helldar** [comment]
@reppair, is there anything funny here that we missed?

**Comment by kristiyanivanovx** [comment]
"Turkey" in Bulgarian means "Турция" (a country), while "turkey" means "пуйка" (type of animal - a bird).
In the translation files for "Turkey" they have used "пуйка" instead of "Турция".

**Comment by andrey-helldar** [comment]
Yandex translator, so that it. I implemented machine translation for them 😂

**Pull request merged**